ClinGen/clincoded
# Connects to #740. Basic implementation of interpretation data comparison

`findDiffKeyValues` is a mixin that holds the functionality for comparing the external data object from a previous interpretation with the newly-generated external data object. Returns a new object with the same keys and sub-keys as the newly-generated external data object, with boolean values indicating whether or not a difference exists with the old external data object.

On the UI end, a simple banner indicating that a change in the data exists. Per-cell/data-item indication of value change will be implemented in #879.
![image](https://cloud.githubusercontent.com/assets/4326866/17492730/ed44a6b2-5d61-11e6-8a1d-b2ec65343cfd.png)

Testing:

1. Create VCI, and begin interpretation
2. Save evaluation on population or predictors-missense tab.
3. Enter the /populations/ or /computational/ object via `#!edit-json` and edit some of the values
4. Reload the form to confirm that the yellow banner exists
5. Re-save the evaluations and confirm that the yellow banner disappears, and that the object in the database has the correct/new data
